keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -666,69 +737,8 @@ Recall that end users cannot usually observe component logs or access metrics.
   - Details: -->
 
 - [x] Other (treat as last resort)
-  - Details:
-
-    1. Launch a cluster with two workload nodes, let's say node1 and node2, both has label key `kubernetes.io/hostname`.
-    2. Add a taint to node1 which ready-to-deploy pods will not tolerate.
-    3. Apply a deployment looks like:
-      ```
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: nginx
-      spec:
-        replicas: 2
-        selector:
-          matchLabels:
-            foo: bar
-        template:
-          metadata:
-            labels:
-              foo: bar
-          spec:
-            restartPolicy: Always
-            containers:
-            - name: nginx
-              image: nginx:1.14.2
-            topologySpreadConstraints:
-              - maxSkew: 1
-                topologyKey: kubernetes.io/hostname
-                whenUnsatisfiable: DoNotSchedule
-                labelSelector:
-                  matchLabels:
-                    foo: bar
-      ```
-    4. One pod will be pending.
-    5. Enable the NodeInclusionPolicy and redeploy the deployment:
-      ```
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: nginx
-      spec:
-        replicas: 2
-        selector:
-          matchLabels:
-            foo: bar
-        template:
-          metadata:
-            labels:
-              foo: bar
-          spec:
-            restartPolicy: Always
-            containers:
-            - name: nginx
-              image: nginx:1.14.2
-            topologySpreadConstraints:
-              - maxSkew: 1
-                topologyKey: kubernetes.io/hostname
-                whenUnsatisfiable: DoNotSchedule
-                NodeTaintsPolicy: Honor
-                labelSelector:
-                  matchLabels:
-                    foo: bar
-      ```
-    6. Both pods will be scheduled successfully.
+  - Details: We can only observe the behaviors based on pod scheduling results.
+  We can follow the steps described at [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy).
```

Comment:
how is this related?
I would just remove this line

Reply:
In case someone want to know the details about how to reproduce this. I'll remove this.